Rocket.Chat/Controllers/Chat/MessagesListViewController.swift

```diff
@@ -29,6 +29,16 @@ extension SubscriptionMentionsResource {
     }
 }
 
+extension SearchMessagesResource {
+    func fetchMessagesFromRealm() -> [Message]? {
+        return raw?["messages"].arrayValue.map { json in
+            let message = Message()
+            message.map(json, realm: Realm.current)
```

Comment:
Maybe save the realm instance before the loop? It may save us some processing and making sure that everything is going to the same database?